inferelator_ng/tfa.py

```diff
@@ -31,12 +31,14 @@ class TFA:
     def __init__(self, prior, exp_mat, exp_mat_halftau):
         self.prior = prior
         self.exp_mat = exp_mat
-        self.exp_mat = exp_mat_halftau
+        self.exp_mat_halftau = exp_mat_halftau
 
     def tfa(self, noself = True, dup_self = True):
         tfwt = self.prior.abs().sum(axis = 0) > 0
         tfwt = tfwt[tfwt].index.tolist()
 
+        dTFs = []
```

Comment:
I agree. I had the same concern before. But Christoph's code didn't take into account the situation where dup_self is False.